routes/api/user/register.js

```diff
@@ -7,8 +7,8 @@ const crypto = require('../../../utils/crypto');
 const router = express.Router();
 
 
-router.post('/', (req, res) => {
-  debug(`register/user: ${JSON.stringify(req.body)}`);
+router.post('/api/user/register', (req, res) => {
```

Comment:
for now, leave strings blank. the module loader in app.js currently will use the pathname in routes/ to determine the associated url. it'll end up as /api/user/register/api/user/register. In the future I'll make it so it uses the routes in the strings and not the path.